app/services/vendor_validator_result_storage.rb

```diff
@@ -0,0 +1,24 @@
+class VendorValidatorResultStorage
+  TTL = Figaro.env.session_timeout_in_minutes.to_i.minutes.seconds.to_i
```

Comment:
This is the TTL for the result value of one job.

So for the 3 steps, they will have 3 separate result objects that will have 3 separate expirations.

When this is asynchronous, the result is created after hitting submit and hearing back from the vendor. So as long as the user's browser takes less than this time to refresh the page (which it definitely will, I think we'll set refresh interval to something like 10-15 seconds) then the controller will pick up the result from redis and use it for what it needs to be used and move on to the next step.